8.js

```diff
@@ -0,0 +1,168 @@
+const real = new Float32Array(2)
+const imag = new Float32Array(2)
+const audioContext = new AudioContext()
+const oscillator1 = audioContext.createOscillator()
+const oscillator2 = audioContext.createOscillator()
+const oscillator3 = audioContext.createOscillator()
+const oscillator4 = audioContext.createOscillator()
+
+real[0] = 0
+imag[0] = 0
+real[1] = 1
+imag[1] = 0
+real[2] = 0.7
+imag[2] = 0
+real[3] = 0.5
+imag[3] = 0
+real[4] = 0.3
+imag[4] = 0
+real[5] = 0.2
+imag[5] = 0
+real[6] = 0.1
+imag[6] = 0
+
+const wave1 = audioContext.createPeriodicWave(real, imag)
+
+oscillator1.setPeriodicWave(wave1)
+oscillator2.setPeriodicWave(wave1)
+oscillator3.setPeriodicWave(wave1)
+oscillator4.setPeriodicWave(wave1)
+
+const gainNode1 = audioContext.createGain()
+const gainNode2 = audioContext.createGain()
+
+oscillator1.connect(gainNode1)
+oscillator2.connect(gainNode2)
+oscillator3.connect(gainNode2)
+oscillator4.connect(gainNode2)
+
+gainNode1.gain.value = 0.5
+gainNode2.gain.value = 0
+
+gainNode1.connect(audioContext.destination)
+gainNode2.connect(audioContext.destination)
+
+const C4 = 440 * 3 / 5
```

Comment:
I'm guessing these are notes?  I'm wondering if there's another representation we could use (array? object?) and a different identifier in order to more clearly communicate the purpose of these `const`s?